spinta/types/inherit/check.py

```diff
@@ -0,0 +1,16 @@
+from spinta import commands
+from spinta.components import Context
+from spinta.exceptions import UndefinedPropertyType
+from spinta.types.datatype import Inherit
+
+
+@commands.check.register(Context, Inherit)
+def check(context: Context, dtype: Inherit) -> None:
+    property = dtype.prop
+    if property.is_reserved():
+        return None
+
+    property_model = property.model
+    property_name = property.name
```

Comment:
necessary?

Reply:
re-used more than once.

Reply:
But it doesn't save any computation time or line length. Maybe rename to just `model` and `name`?

Reply:
optional